dts/bindings/sensor/ti,ina23x.yaml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright 2021 The Chromium OS Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+description: |
+    TI INA230 and INA231 Bidirectional Current and Power Monitor.
+    The <include/dt-bindings/sensor/ina23x.h> file should be included
+    in the DeviceTree and it provides Macros that can be used for
+    initializing the Configuration register.
+
+compatible: "ti,ina23x"
+
+include: i2c-device.yaml
+
+properties:
+    config:
+      type: int
+      required: true
+      description: Configuration register
+
+    current_lsb:
+      type: int
+      required: false
+      default: 1
+      # default of 1mA for current LSB
+      description: |
+        Value of Current LSB in milliamps. When set to 1mA
+        Current is read in mA, Bus Voltage in mV, Shunt
+        Voltage in uV, and Power in mW. Any other value
+        results in Current, Voltage, and Power registers
+        being read in counts.
```

Comment:
You need to justify the use of `default` as described in https://docs.zephyrproject.org/latest/guides/dts/bindings.html#default-values-for-properties. E.g. if this is the power-on-reset value, then you must document this fact in the binding description.

Reply:
The default value was there to make life easier for the user but after ready the documentation, it seems better to let the user make a conscious decision about setting the value.